ipld/src/util.rs

```diff
@@ -78,3 +80,70 @@ where
 
     Ok(())
 }
+
+pub const DEFAULT_RECENT_ROOTS: ChainEpoch = 2000;
```

Comment:
This is not a `ChainEpoch`, it's a delta.

Reply:
changed to i64